### DISCUSSION TITLE
GitLab Webhook on disabled repo?

### DISCUSSION THREAD

**Discussion by sichapman** [mend/renovate-ce-ee#590] [Q&A] [answered]
Loving the new webhook creation, was one of the things that's prompted me to check CE out.

But if my renovate.json is 

```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "enabled": false
}
```

would you expect a webhook to be created or not? Because it currently is - seems a bit of a waste

**Comment by rarkins** [comment, marked as answer]
It's expected. At this point the complexity it would add to sync webhooks to repo config is not worth it considering the minor efficiency gain.
